build.sbt

```diff
@@ -613,9 +613,10 @@ lazy val enumeratumSlick =
       version            := "1.7.5-SNAPSHOT",
       crossScalaVersions := scalaVersionsAll,
       libraryDependencies ++= Seq(
-        ("com.typesafe.slick" %% "slick" % "3.4.1").cross(CrossVersion.for3Use2_13),
+        ("com.typesafe.slick" %% "slick" % "3.5.0-M4"),
```

Comment:
Please note this depends on a milestone version, so we can wait a bit

Reply:
I see we'er at M5 now; let's wait until we get to a stable release.